tests/user_timers_test.go

```diff
@@ -126,85 +125,3 @@ func (s *UserTimersTestSuite) TestUserTimers_Sequential() {
 	s.NoError(err)
 	s.True(workflowComplete)
 }
-
-func (s *UserTimersTestSuite) TestUserTimers_CapDuration() {
```

Comment:
Why did you removed it? Functionality shouldn't change.

Reply:
@yycptt had left a comment that the duration cap logic can be removed and we can rely on logic in mutable state to drop long duration timers: https://github.com/temporalio/temporal/pull/6631/files#diff-70d155a6ca0ad36ef8abc4e120ce84c2f321deb783a0c98789263320e799e140L48

This seemed like a good time to remove it since I was already updating the validation logic.